packages/hint-css-prefix-order/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "dependencies": {
+    "postcss": "^7.0.14"
+  },
+  "description": "Ensure vendor-prefixed versions of a CSS property are listed before the unprefixed version.",
+  "devDependencies": {
+    "@hint/parser-css": "^2.0.3",
+    "@hint/utils-tests-helpers": "^2.0.3",
+    "@types/debug": "0.0.31",
+    "@types/node": "8.0.14",
+    "ava": "^1.2.0",
+    "cpx": "^1.5.0",
+    "eslint": "^5.12.1",
+    "eslint-plugin-markdown": "^1.0.0",
+    "eslint-plugin-typescript": "^0.14.0",
+    "hint": "^4.3.1",
+    "markdownlint-cli": "^0.13.0",
+    "npm-link-check": "^3.0.0",
+    "npm-run-all": "^4.1.5",
+    "nyc": "^13.1.0",
+    "postcss": "^7.0.14",
+    "rimraf": "^2.6.3",
+    "typescript": "^3.3.1",
+    "@typescript-eslint/parser": "^1.2.0"
+  },
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "dist/src"
+  ],
+  "homepage": "https://webhint.io/",
+  "keywords": [
+    "hint",
+    "hint",
+    "css-prefix-order",
+    "css-prefix-order-hint"
+  ],
+  "license": "Apache-2.0",
+  "main": "dist/src/hint.js",
+  "name": "@hint/hint-css-prefix-order",
+  "nyc": {
+    "extends": "../../.nycrc"
+  },
+  "peerDependencies": {
+    "@hint/parser-css": "^2.0.3",
+    "hint": "^4.3.1"
+  },
+  "private": true,
+  "repository": "webhintio/hint",
+  "scripts": {
+    "build": "npm run clean && npm-run-all build:*",
```

Comment:
We no longer do the `clean` on build (and we should update `create-hint` for this)